README.md

```diff
@@ -1 +1,193 @@
-# Philemon-portfolio
\ No newline at end of file
+# Philemon-portfolio
+<a name="readme-top"></a>
+
+<div align="center">
+
+ # Kieng Philemon
+
+  <h3><b>Kieng Philemon portfolio templates</b></h3>
+
+</div>
+
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# Hello Microverse
```

Comment:
- [ ] I will suggest you ensure that the contents of your README are related to the current project you're working on so users aren't confused or misled when they visit your repository :wink: